### PR TITLE
blog(embertimes129): 3 New RFCs

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2019-12-20-the-ember-times-issue-129.md [hunk 1]
```diff
@@ -37,13 +38,17 @@ Qonto wrapped up the code in a small addon, [`ember-autofocus-modifier`](https:/
...
-<consider adding some bold to your paragraph>
+[Miguel Camba @cibernox](https://github.com/cibernox) opened a slew of RFCs around adding new built-in operators to Ember templates. The idea comes from functionality provided in [ember-truth-helpers](https://github.com/jmurphyau/ember-truth-helpers) and is a subset of his [other RFC](https://github.com/emberjs/rfcs/pull/388) to add basic helpers to Ember templates.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The three template helper RFCs Miguel proposed are the following:
+
+- [Adding Equality Operators](https://github.com/emberjs/rfcs/pull/560) such as the `{{eq}}` and `{{neq}}` helpers
+- [Adding Numeric Comparison Operators](https://github.com/emberjs/rfcs/pull/561), specifically the `{{lt}}` and `{{gt}}` helpers
+- [Adding Logical Operators](https://github.com/emberjs/rfcs/pull/562) which are the {{and}}, {{or}} and {{not}} helpers
```
Comment (jayjayjpg) [inline source/2019-12-20-the-ember-times-issue-129.md]: What do you think about wrapping the code here as well?

```suggestion
- [Adding Logical Operators](https://github.com/emberjs/rfcs/pull/562) which are the `{{and}}`, `{{or}}` and `{{not}}` helpers
```

File: source/2019-12-20-the-ember-times-issue-129.md [hunk 2]
```diff
@@ -37,13 +38,17 @@ Qonto wrapped up the code in a small addon, [`ember-autofocus-modifier`](https:/
...
+
+Help get this of the ground by adding your input to these changes today!
```
Comment (jayjayjpg) [inline source/2019-12-20-the-ember-times-issue-129.md]: I think it'd be

```suggestion
Help get this off the ground by adding your input to these changes today!
```

File: source/2019-12-20-the-ember-times-issue-129.md [hunk 3]
```diff
@@ -37,13 +38,17 @@ Qonto wrapped up the code in a small addon, [`ember-autofocus-modifier`](https:/
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## 3 New RFCs To Add New Template Helpers 3️⃣
```
Comment (jayjayjpg) [inline source/2019-12-20-the-ember-times-issue-129.md]: The goodbitsing newsletter tool currently expects each title to also have an url present. For example, we could link the RFC repos with all of Miguel's relevant template helper PRs that are mentioned in this section. What do you think?

```suggestion
## [3 New RFCs To Add New Template Helpers 3️⃣](https://github.com/emberjs/rfcs/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+author%3Acibernox+templates)
```

**Review by jayjayjpg — APPROVED** [review]
This reads great and I left a few more suggestions - lmk what you think! Otherwise, thank you for another great :sparkles: (and on top of that super early!) writeup this week :rainbow:

File: source/2019-12-20-the-ember-times-issue-129.md (hunk 3 above)
Comment (chrisrng) [reply source/2019-12-20-the-ember-times-issue-129.md]: Oh this is awesome didnt think about this!

**Comment by chrisrng** [comment]
@jessica-jordan omg thanks! I totally feel asleep on the wheel (feeling a bit under the weather) and forgot to `code` all the template helpers

**Pull request merged**